### ISSUE TITLE
Example from README not working

### ISSUE THREAD

**Issue by fepegar** [bruce-szalwinski/mkdocs-typer#18]
Hi, and thanks for this library!

I'm trying the example from the README, but I'm getting the following error:

```python-traceback
mkdocs_typer._exceptions.MkDocsTyperException: 'cli' must be a 'typer.main.Typer' object, got <class 'function'>
```
If I use `:command: my_app` (instead of `cli`), I get no error, but I see a big `my_app` in the docs:

<img width="371" alt="Screenshot 2023-11-27 at 01 52 12" src="https://github.com/bruce-szalwinski/mkdocs-typer/assets/12688084/64974e6a-e11f-4a06-92c6-288e60480091">

1. Is there a bug?
1. Am I doing something wrong?
1. Is it possible to hide that `my_app` header? I tried playing with `:depth:` with no luck.

**Comment by multimeric** [comment]
Indeed, this seems to be broken. My guess is that the `typer` API changed such that `@command` no longer returns a `typer` object and now just returns the unchanged function. This package should be able to work with the `app` object but as you noted, that's also broken.

I found a workaround that I'm quite happy with, though:
1. `pip install mkdocs-click`
2. Use `mkdocs-click` in your `mkdocs.yml`:
```diff
markdown_extensions:
-- mkdocs-typer
+- mkdocs-click
```
3. Create a click app in your Python module:
```diff
+ from typer.main import get_command
app = Typer()
+ click_app = get_command(app)
```
4. Update your markdown docs:
```diff
-::: mkdocs-typer
-    :module: my_pkg.__main__
-    :command: app
+::: mkdocs-click
+    :module: my_pkg.__main__
+    :command: click_app
```

**Comment by fepegar** [comment]
Thanks for your reply, @multimeric!
